src/views/Links/index.jsx

```diff
@@ -19,17 +19,30 @@ import FeedbackIcon from '@mui/icons-material/Feedback';
 import SafetyDividerIcon from '@mui/icons-material/SafetyDivider';
 import SpaIcon from '@mui/icons-material/Spa';
 import PsychologyIcon from '@mui/icons-material/Psychology';
+import LocalPoliceIcon from '@mui/icons-material/LocalPolice';
+import CallIcon from '@mui/icons-material/Call';
+import CategoryIcon from '@mui/icons-material/Category';
+import ReportIcon from '@mui/icons-material/Report';
+// import EmergencyIcon from '@mui/icons-material/Emergency';
 import styles from '/src/views/Links/Links.module.scss';
 import { useNavigate } from 'react-router';
+import { styled } from '@mui/material/styles';
 
 const Links = () => {
   const navigate = useNavigate();
 
+  const CardContentWithPadding = styled(CardContent)(`
+    padding: 8px;
+    &:last-child {
+      padding-bottom: 8px;
+    }
+  `);
+
   // Academics Resources UI
   const academicsCard = (
     <Card>
       <CardHeader title={`Academics`} className="gc360_header" />
-      <CardContent>
+      <CardContentWithPadding>
```

Comment:
Probably use CSS classes here, instead of a custom styled component if possible, just to be more in line with how the rest of 360 approaches styling.  And for the other cards too